### PR TITLE
Icon: Update Icons 

### PR DESCRIPTION
**Pull request by rlingineni**
# Pull Request Instructions
Apologies for the large set of changes. This PR addresses the latest Icons Request as exported from Figma. This ensures a consistent output. The PR appears noisy, but it's only about 5 actual changes.

A majority of changes are internal Android SVG optimizations - as outputted by @protodave's script with no change on the final look. Fixing this once will ensure consistent output from the script going forward.

## Diff
Interactive differences in the PR are available to see here.

[VR Icons](https://deploy-preview-3690--gestalt.netlify.app/vr-icons-diff.html)
[Classic Icons](https://deploy-preview-3690--gestalt.netlify.app/icons-diff.html)

## New Icons Introduced

[Classic](https://deploy-preview-3690--gestalt.netlify.app/icons-diff.html#new-icons)
[VR](https://deploy-preview-3690--gestalt.netlify.app/vr-icons-diff.html#new-icons)

- building-business.svg
- birthday-cake
- link-broken

## Breaking Deprecations/Renames
|Old Name|New Name|
|----|---|
`view-type-space` | `view-type-sparse`
`x` | `x-social`

### Links

- [Jira](https://jira.pinadmin.com/browse/GESTALT-XXXX)
- [TDD](link to Paper doc)
- [Figma](link to Figma file)

### Checklist

- [ ] Added unit tests
- [ ] Added documentation + accessibility tests
- [ ] Verified accessibility: keyboard & screen reader interaction
- [ ] Checked dark mode, responsiveness, and right-to-left support
- [ ] Checked stakeholder feedback (e.g. Gestalt designers, relevant feature teams)

### CODE REVIEW

File: packages/gestalt/src/icons/index.ts [hunk 1]
```diff
@@ -52,12 +52,14 @@ import audioMix from './audio-mix.svg';
...
 import bell from './bell.svg';
 import bellFill from './bell-fill.svg';
+import birthdayCake from './birthday-cake.svg';
 import board from './board.svg';
 import boardStickerLarge from './board-sticker-large.svg';
 import boardStickerSmall from './board-sticker-small.svg';
 import border from './border.svg';
 import borderShadow from './border-shadow.svg';
 import briefcase from './briefcase.svg';
+import buildingBusiness from './building-business.svg';
```
Comment (rlingineni) [inline packages/gestalt/src/icons/index.ts:62]: alphabetizing

**Comment by AlbertCarreras** [comment]
Screenshot by Dropbox Capture

**Comment by protodave** [comment]
Reviewed the web Classic icon svg changes and the [visual diff.](https://deploy-preview-3690--gestalt.netlify.app/icons-diff.html) Couple of notes:

* `microphone`
Clipping on the top
Screenshot 2024-07-18 at 5 07 11 PM

* `pinterest`
Left side of the circle has drifted a little causing a slight clipping. (this is also happening in the VR set)
Screenshot 2024-07-18 at 5 21 46 PM

* `search`, `search-fill`
Handles are clipping a little
Screenshot 2024-07-18 at 5 11 47 PM

* `view-type-space`
This is not found in the Figma file. If this is intentional and we want to remove it then this PR should also delete the file `view-type-space.svg`. Note: This icon does still exist in the VR Figma file.
Screenshot 2024-07-18 at 5 17 27 PM

* `view-type-sparce`, `view-type-sparse`
The bulk export seems to have found two separate icons for this in FIgma, but I only see `-sparse` added in this PR.
Screenshot 2024-07-18 at 5 30 05 PM

**Comment by rlingineni** [comment]
Abandoning - will make a new one

**Pull request closed without merging**